qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -309,6 +330,10 @@ def shutdown():
     pass
 
 
+if qtutils.version_check('5.8'):
+    MAPPINGS['spellcheck.languages'] = DictionaryLanguageSetter()
```

Comment:
You now never call `setSpellCheckEnabled(True)` on the profiles, you should probably do so on Qt 5.8 in `_init_profiles` (but like discussed, without any setting).

Curiosly, on e.g. https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_textarea I still see the URL underlined when I activate the field, but as soon as I start typing it's gone when not enabling spell checking on the profile (I did so via `:debug-console`).